Gopkg.toml

```diff
@@ -24,7 +24,7 @@
 
 [[constraint]]
   name = "k8s.io/helm"
-  version = "2.8.0"
+  version = "2.9.0"
```

Comment:
2.9.0 has some bugs that break new lines in yaml. 2.9.1 is out now which fixes this. We should skip 2.9.0 and go to the .1 release.

Reply:
done!